README.md

```diff
@@ -1,13 +1,34 @@
 # snowbox [![Build Status](https://travis-ci.com/algotech/snowbox.svg?branch=master)](https://travis-ci.com/algotech/snowbox)[![codecov](https://codecov.io/gh/algotech/snowbox/branch/master/graph/badge.svg)](https://codecov.io/gh/algotech/snowbox)
 
-## Install
+Opinionated Redux abstractions for faster development.
```

Comment:
@lgnstn I suggest to add the following sentence as well: `Although it is created explicitly for React and React native, it works with any JavaScript framework that uses Redux.`. What do you think?

Reply:
It is just partially true because some of its features use Hooks, which are React specific.